package.json

```diff
@@ -134,6 +137,7 @@
     "tslib": "^1.9.3",
     "tslint": "~5.11.0",
     "typescript": "~3.2.2",
+    "webpack": "^4.0.0",
     "webpack-cli": "^3.1.2"
   }
 }
```

Comment:
There are some `devDependencies` that can be upgraded. You can use VSCode's built-in functionality to hover over a dependency version to see the most recent one. Please do this only for `devDependencies`